### DISCUSSION TITLE
Capacitor 2.4.0, LocalNotification feature

### DISCUSSION THREAD

**Discussion by MdAman02** [ionic-team/capacitor#3737] [Q&A]
Any way i can stop the App coming to foreground when a localnotification action is tapped?
I am using this interface: (https://capacitorjs.com/docs/apis/local-notifications#addlistener)

**Comment by leo-petrucci** [comment]
I don't think that's possible, it's how notifications are built to work.
What's the use case if you don't mind me asking?

**Comment by aparajita** [comment]
When a user specifically taps a notification, they are signaling they want to see the details of the notification, as opposed to swiping to dismiss it. This is standard system behavior, and unless you can come up with a better interaction model, should not be overridden, otherwise you risk confusing your users.
